Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/TimeView/TimeViewPlugin.cpp

```diff
@@ -14,11 +15,12 @@
 #include "TimeInfoWidget.h"
 #include "TimeViewToolBar.h"
 
-#include "../MotionWindow/MotionWindowPlugin.h"
-#include "../MotionWindow/MotionListWindow.h"
-#include "../MotionSetsWindow/MotionSetsWindowPlugin.h"
-#include "../MotionEvents/MotionEventsPlugin.h"
-#include "../MotionEvents/MotionEventPresetsWidget.h"
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionWindowPlugin.h>
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionListWindow.h>
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionSetsWindow/MotionSetsWindowPlugin.h>
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionEvents/MotionEventsPlugin.h>
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionEvents/MotionEventPresetsWidget.h>
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionEvents/MotionEventWidget.h>
 
 #include "../../../../EMStudioSDK/Source/EMStudioManager.h"
```

Comment:
Great job for fixing the above line!. Maybe also fix these two line also?

Reply:
Good call, done